libert_util/src/log.c

```diff
@@ -128,6 +128,17 @@ bool log_include_message(const log_type *logh , int message_level) {
     return false;
 }
 
+/**
+ * Adds a string to the log if message_level is below the threshold
+ * @param logh The log  to add to
```

Comment:
My IDE adds this doxygen formating, but I see that they are not present elsewhere in the code.